tests/test_downloader_handlers.py

```diff
@@ -177,6 +177,13 @@ def render(self, request):
         return request.content.read()
 
 
+class LargeChunkedFileResource(resource.Resource):
+    def render(self, request):
+        for i in range(1024):
+            request.write(b"\x00" * 1024)
+        return b""
```

Comment:
I think it should be `request.finish()` instead of `return b""` even if it doesn't make a difference for the test case.